src/components/Snapshots.vue

```diff
@@ -575,6 +725,296 @@ export default class Snapshots extends Vue {
     return results;
   }
 
+  get manualScalebarSettingsLength(): number {
+    if (this.manualScalebarSettings) {
+      return this.manualScalebarSettings.length;
+    }
+    return 1.0;
```

Comment:
This (and also the following getters) could be shortened with: `return this.manualScalebar?.length || 1.0`